duckduckgo/iOS
# Replace windows waitlist with download link in mobile settings

Task/Issue URL: https://app.asana.com/0/1204186595873227/1204514556221984/f

**Description**: It will replace the Windows waitlist flow with one that matches the current Open Beta Mac implementation, pointing to the Windows download page when the related config flag is on.



**Steps to test this PR**:
1. Go to settings and to DuckDuckGo Windows App since the feature is disabled remotely check that the page is the waitlist page (depending on your state in the queue etc)
2. Go to WindowsBrowserWaitlist.swift and on line 52 change isWaitlistRemoved  to true
3. Test that the DuckDuckGo App for windows page shows the copy and feature indicated in https://app.asana.com/0/0/1204562778402691/f



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
